src/sfctl/custom_app.py

```diff
@@ -166,7 +166,7 @@ def parse_app_params(formatted_params):
 
     res = []
     for k in formatted_params:
-        param = ApplicationParameter(k, formatted_params[k])
+        param = ApplicationParameter(key=k, value=formatted_params[k])
```

Comment:
Why do we need to do this now? Did something change in the SDK so we have all these optional named arguments?

Reply:
Yes. The SDK was changed because of the new auto-rest